src/components/dialogs/parameters/dynamicsimulation/solver-parameters.js

```diff
@@ -5,72 +5,107 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
 
+import yup from '../../../utils/yup-config';
 import { Grid } from '@mui/material';
-import { makeComponentsFor, TYPES } from '../util/make-component-utils';
-import { useCallback, useMemo } from 'react';
-import IdaSolverParameters from './solver/ida-solver-parameters';
-import SimplifiedSolverParameters from './solver/simplified-solver-parameters';
+import { useEffect, useMemo } from 'react';
+import { useWatch } from 'react-hook-form';
+import { SelectInput } from '@gridsuite/commons-ui';
+import { makeComponents } from '../util/make-component-utils';
+import IdaSolverParameters, {
+    getFormSchema as getIdaFormSchema,
+} from './solver/ida-solver-parameters';
+import SimplifiedSolverParameters, {
+    getFormSchema as getSimplifiedFormSchema,
+} from './solver/simplified-solver-parameters';
+import { TabPanel } from '../parameters';
 
-const SOLVER_TYPES = {
+export const SOLVER_TYPES = {
     IDA: 'IDA',
     SIM: 'SIM',
 };
 
-const SolverParameters = ({ solver, onUpdateSolver }) => {
-    const { solverId, solvers } = solver ?? {};
+export const SOLVER_ID = 'solverId';
```

Comment:
In my opinion, using a common file is better in case in which we have different  sibling components sharing the same information. In my case, I have decomposed SolverParameter from the parent DynamicSimulationParameter component for the reason of code isolation and make child components more independent, and not need to share information.

Reply:
After the merge of https://github.com/gridsuite/gridstudy-app/pull/1679, i have created a new file which shares common parameters between two solvers.

https://github.com/gridsuite/gridstudy-app/pull/1613/files#diff-b450acac854ed6d2eef8c7db37e5759616ea662c5ce1f1dbab2078e61ab37c2c